src/js/organisms/notification-bar.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Handles events related to the notification bar
+ *
+ * @class NotificationBar
+ */
+
+module.exports = class NotificationBar {
+  static init() {
+    const dissmissButtons = document.querySelectorAll(
+      '.mzp-c-notification-bar-button',
+    );
+    for (let i = 0; i < dissmissButtons.length; i += 1) {
+      dissmissButtons[i].addEventListener(
+        'click',
+        function handler(e) {
+          e.currentTarget.parentNode.remove();
+        },
+        false,
+      );
+    }
+  }
+};
```

Comment:
This class definition follows the pattern of the other "bootstrapping" JS in the project

Reply:
The code here is simply to handle the dismissal button in a generic way. There is a separate class that deals with behaviour specific to the task-completion-survey use of the notification.